.github/workflows/cd_ios.yml

```diff
@@ -0,0 +1,60 @@
+name: Interactive Player RN - iOS Build and Distribution
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
+
+jobs:
+  Build-And-Deploy-iOS-App-To-Appstore:
+    runs-on: macos-13
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - name: Install modules
+        run: yarn install
+
+      - name: Create internal environment
+        env:
+          ENV: ${{secrets.ENV}}
+        run: |
+          echo $ENV > .env
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1.2
+
+      - name: Setup Fastlane
+        run: |
+          cd ios
+          bundle install
+          bundle exec pod install
```

Comment:
`gem install bundler` and `bundle install` can be moved here, so we don't need it in the step below?